karma.conf.js

```diff
@@ -20,7 +20,7 @@ module.exports = function (config) {
       'karma-sourcemap-loader'
     ],
 
-    browsers: ['PhantomJS'],
+    browsers: process.env.CI === true ? ['PhantomJS'] : [],
```

Comment:
Then that forces people to always specify a browser. I suppose this can be wrapped in the npm script so there's still a convenient shorthand.